### ISSUE TITLE
DTSXU support / volume balancing setting

### ISSUE THREAD

**Issue by Osahashi** [pal1000/Realtek-UAD-generic#100]
I bought a new motherboard with new audio chip, my new Asus PRIME Z590-A with the Realtek S1220A offers DTSXU support, but the driver is a problem with the whole thing.

Asus offers a driver version 6.0.9244.1 with DTSXU, but it lacks support for volume balancing.
The driver from you pal, does not offer DTSXU support, but has volume balancing.

I have already tried several things, I installed the Asus driver and then manually where it could replace the drivers with the UAD driver, but then DTSXU disappears and no volume balancing appears either. The UAD driver is more recent, so I don't know if it makes more sense to install it first and then pull the Asus driver over it.

DTSXU would be nice, but in the end the volume balance is more important to me and I really need that.

The Asus driver can be found here (177.08 MBytes): https://dlcdnets.asus.com/pub/ASUS/mb/01AUDIO/DRV_Audio_RTK_UWD_DTSXU_TP_W11_64_V6092441_20211027R.zip

Anyone have an idea how to get DTSXU into the UAD driver?

**Comment by epman100** [comment]
try either [v6.0.9254.1](https://dlcdnets.asus.com/pub/ASUS/mb/01AUDIO/DRV_Audio_RTK_SZ_600_DAP_W11_64_V6092541_20211117R.zip) or [v6.0.9282.1](https://dlcdnets.asus.com/pub/ASUS/mb/LGA1700/M6621/Realtek_Audio_Driver_V6.0.9282.1_WIN10_WIN11_64-bit.zip) from ASUS

**Comment by Osahashi** [comment]
Got it now, thanks for the link.

I updated all drivers to the ones from the v6.0.9254.1 package, strangely it shows me 6.0.9282.1 as the installed driver in RAC Center.
Then I manually added all the DTSXU drivers from the package from my motherboard with the device manager, only this way it showed me DTS Audio Effects Components, before it was not there. Then I was finally able to install the DTSXUltra app from the driver package. 

Now I have a homemade driver, but DTSXU works and volume balancing works as well.
